testsuite/integration-arquillian/servers/auth-server/services/testsuite-providers/src/main/java/org/keycloak/testsuite/federation/BackwardsCompatibilityUserStorage.java

```diff
@@ -334,7 +334,7 @@ public int getUsersCount(RealmModel realm) {
     }
 
     @Override
-    public Stream<UserModel> searchForUserStream(String search, RealmModel realm, int firstResult, int maxResults) {
+    public Stream<UserModel> searchForUserStream(String search, RealmModel realm, Integer firstResult, Integer maxResults) {
```

Comment:
Are you sure? I changed int -> Integer for all stream variant, hence we won't break backward compatibility, because stream variants are new and old implementations won't be affected. However, since all implementations in our code implement XXX.Stream variants of interfaces we should change them. Does it make sense? Or I should change to int even new methods.

Reply:
Ah, I missed that this is the stream variant, you're right. Sorry for false alarm!